packages/components/src/local/organisms/setting-channels/index.tsx

```diff
@@ -77,6 +77,7 @@ export const SettingChannels: React.FC<PropTypes> = ({
   const [participantKey, confirmRemoveParticipant] = useState<number>(-1)
   const [postRemoveActionConfirmed, setPostRemoveActionConfirmed] = useState<boolean>(false)
   const [problems, setProblems] = useState<string>('')
+  const [isChecked, setChecked] = useState<boolean>(false)
```

Comment:
'Checked' is a bit vague. Could we use 'isChatAuthorHidden` and `setChatAuthorHidden` instead please?